### ISSUE TITLE
unable to run baseline profile when flybuy dependency is in project

### ISSUE THREAD

**Issue by hatched-MaciejPrzybylski** [RadiusNetworks/flybuy-android#25]
if you try to run baseline profile generator following exception is thrown during building

```
- Variant 'defaultReleaseVariantSdkJavaDocPublication' capability com.radiusnetworks.flybuy:pickup:2.12.0 declares a component for use during runtime:
              - Unmatched attributes:
                  - Doesn't say anything about com.android.build.api.attributes.AgpVersionAttr (required '8.4.1')
                  - Doesn't say anything about com.android.build.api.attributes.BuildTypeAttr (required 'nonMinifiedRelease')
                  - Doesn't say anything about com.android.build.api.attributes.ProductFlavor:env (required 'staging')
                  - Provides attribute 'com.android.build.api.attributes.ProductFlavor:flybuy' with value 'default' but the consumer didn't ask for it
                  - Provides documentation but the consumer didn't ask for it
                  - Provides its dependencies declared externally but the consumer didn't ask for it
                  - Provides javadocs but the consumer didn't ask for it
                  - Provides release status but the consumer didn't ask for it
```

**Comment by botnerd** [comment]
Can you please provide details on which version of the SDK you are using, what android SDK version you are targeting, and the android device information?

**Comment by hatched-MaciejPrzybylski** [comment]
flybuy version: 2.12.0 
android target sdk:  33

**Comment by botnerd** [comment]
SDK version 2.10.0 and higher requiring targeting Android 14. Please see [these upgrade instructions](https://www.radiusnetworks.com/developers/flybuy/#/sdk-2.0/android-14) as there are some small changes that need to be addressed.

**Comment by hatched-mariusz** [comment]
issue can be closed for version 2.13.0 android target SDK 34.
